.codeclimate.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
+exclude_patterns:
+- "node_modules"
+- "middlewares/validations/validations.js"
+- "controllers/profile.js"
```

Comment:
kindly, why are you excluding your controller and middlewares to bypass `codeclimate`